### ISSUE TITLE
skr 1.4 x and y min endstop wont trigger

### ISSUE THREAD

**Issue by nevertyam** [bigtreetech/BIGTREETECH-SKR-V1.3#181]
i havw the ground to signal wires connected butcant get a trigger when i do a m119

**Comment by nevertyam** [comment]
cutoff the pin for tmc 2209 if not using stallguard fixed this...

**Comment by Gerggg** [comment]
Any news ?

**Comment by nevertyam** [comment]
I was using 2209 driver in uart and without stall guard. So the y and x weren’t triggering what so ever. The fix was to cutoff that diag pin just like the instructions

**Comment by Gerggg** [comment]
Cutting a pin does not seem to me to be a good solution ... especially with a SKR 1.3 I do not have the problem.
I'm going to look for a less definitive solution.
thank you

**Comment by Gerggg** [comment]
I'm going to see if on the Marlin side, they manage to find a sofware solution

**Comment by nevertyam** [comment]
Yeah I would agree. They made the 1.4 and 1.3 issues on the same github. But the title says 1.4 and so this fix is for the 1.4 board with tmc2209 only. So it’s pretty specific.

For you however, do a m119 command. If it’s already triggered without physically being triggered, that’s an easy fix, just invert it in firmware.

**Comment by Gerggg** [comment]
I do not see how, what is the option in Marlin?

**Comment by nevertyam** [comment]
do a m119 command. If it’s already triggered without physically being triggered, that’s an easy fix, just invert it in firmware. 

Look for X_MIN_ENDSTOP_INVERTING false
And set to true IF your getting a triggered command on a m119

**Comment by Toby00** [comment]
I have the same problem whit tmc2209 i have cut the pin off.
The endstops are not working i have tmc 2209 v1.2

**Comment by lijinlin19970813** [comment]
> I was using 2209 driver in uart and without stall guard. So the y and x weren’t triggering what so ever. The fix was to cutoff that diag pin just like the instructions

In fact, it is not necessary to cut the pin, you only need to bend the pin so that it does not insert into the hole.

**Comment by c3D-Dan** [comment]
Sounds like your signal voltage is too high.  Please check your 3.3v Rail is actually measuring 3.3v .   If it's measuring higher, replace your AM1117 LDO Regulator

**Comment by nicobits46** [comment]
hi i´m having the same problem with a skr 1.4 turbo.
x and y won´t home, i cut off the diag pin because i want to use the endstops but i can{t get them to home.
the endstops work perfect doing an M119 . it´s making me crazy thank you!!!

**Comment by rkarlsba** [comment]
This is the first time I've read a manual that tells me to destroy my hardware to fix it. Perhaps it should be better to fix this properly?

**Comment by c3D-Dan** [comment]
You shouldn't have to.  The SKR boards are poorly designed. Replace it with a GTR. In the GTR they fixed the fuckups. Get a GTR. They're better boards and MUCH more reliable.
